### PR TITLE
Added tests for creating/deleting * domain.

### PR DESCRIPTION
**Pull request by Pchelolo**
1. Fixed a bug with query parameter validation
2. Removed wrong test regarding ordering of the query parameters. The order of the object properties is not specified anyway.
3. Added tests for the new feature of creating the keyspaces/tables for many domains.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "restbase-mod-table-spec",
-  "version": "1.1.2",
+  "version": "1.1.3",
```
Comment (d00rman) [inline package.json]: This should probably be 1.2.0

**Review by d00rman — COMMENTED** [review]
For some reason, this PR has conflicts with master?

**Comment by Pchelolo** [comment]
Rebased, bumped. All done.

**Comment by d00rman** [comment]
LGTM, will merge/release once v1.1.3 has been tagged and published.

**Comment by Pchelolo** [comment]
Done.

**Comment by Pchelolo** [comment]
Published as restbase-mod-table-spec@1.2.0

**Pull request merged**